pifi/videoprocessor.py

```diff
@@ -318,13 +335,22 @@ def __get_process_and_play_vid_cmd(self, ffmpeg_to_python_fifo_name):
             maybe_save_video_tee = shlex.quote(temp_video_save_path) + ' '
             maybe_mv_saved_video_cmd = '&& mv ' + shlex.quote(temp_video_save_path) + ' ' + shlex.quote(video_save_path)
 
+        fps_cmd = f'>( ffprobe -v 0 -of csv=p=0 -select_streams v:0 -show_entries stream=r_frame_rate - >{fps_fifo_name} ) '
```

Comment:
nit: I'd think we can probably write to `fps_fifo_name` directly rather than writing to stdout and redirecting to the file :)

Reply:
What do you mean? The `-` here is instructing `ffprobe` to read from stdin, not to write to stdout. I don't see a way to ask `ffprobe` to write directly to a file - it seems to always output to `stdout`.

Reply:
Oh, I've misunderstood. Please ignore!